src/frontend/components/Navbar/Navbar.css

```diff
@@ -5,13 +5,28 @@
 }
 
 .header-nav {
-  margin-top: 1rem;
   margin: 0;
-  background: none;
+  background-color: var(--body-bg-color);
   position: fixed;
-  width: 100vw;
+  top: 0;
+  /* width: 100vw; */
```

Comment:
If not necessary do remove this comment.

Reply:
Missed this. I'll do it.